edx_oauth2_provider/tests/test_create_oauth2_client.py

```diff
@@ -96,16 +98,15 @@ def test_client_creation(self, args, options):
         self.assert_client_created(args, options)
 
     @ddt.data(
-        ((URL, REDIRECT_URI), 'too few arguments'),
```

Comment:
Reason of change. 
python 2.7 Error : too few arguments.
python 3.6 Error : Argument is required.

Reply:
Please add this context as a comment on this line